experimental/sql/src/ibis_frontend.py

```diff
@@ -71,6 +78,17 @@ def visit(  #type: ignore
   raise Exception(f"Unknown stringcolumn: {type(op)}")
 
 
+@dispatch(ibis.expr.types.IntegerColumn)
+def visit(  #type: ignore
+    intColumn: ibis.expr.types.IntegerColumn) -> Operation:
+  op = intColumn.op()
+  if isinstance(op, gen_types.TableColumn):
+    table = Region.from_operation_list([visit(op.table)])
+    new_op = id.TableColumn.get(table, op.name)
+    return new_op
+  raise Exception(f"Unknown stringcolumn: {type(op)}")
```

Comment:
What is this class for? Are you using it in this PR already? (If not, separate it?)

Reply:
This class is very similar to the handling of stringcolumns, but occurs when the column type is an int and not a string. So the class is actually used in the PR, since I enriched the example database, but it doesn't really have something to do with the topic of the PR. I guess I can move this out to a different PR if you want.